### ISSUE TITLE
Focus rings should be 1px "light" inside and 2px "dark" outside. They are only 2px dark outside color.

### ISSUE THREAD

**Issue by miroslavstastny** [microsoft/fluentui#19092]
Focus rings should be 1px "light" inside and 2px "dark" outside. They are only 2px dark outside color.

[Juraj] I don't think we should change the focus indicator in v9, unless there was a decision I am not aware of

@bsunderhus

**Comment by khmakoto** [comment]
Should we maybe ask design what would be the correct thing to do here?
